src/Forms/TextareaField.php

```diff
@@ -106,6 +116,25 @@ public function getColumns()
         return $this->cols;
     }
 
+    /**
+     * @param int $maxLength
+     * @return $this
+     */
+    public function setMaxLength($maxLength)
+    {
+        $this->maxLength = $maxLength;
+
+        return $this;
+    }
+
+    /**
+     * @return null|int
+     */
+    public function getMaxLength()
```

Comment:
TextField adds this to `getAttributes`. Shouldn't be do this in TextareaField also?